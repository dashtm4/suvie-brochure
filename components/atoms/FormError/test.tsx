import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { FormError } from '.'

describe('<FormError />', () => {
  it('should has correct props', () => {
    const name = 'test'

    const wrapper = shallow(<FormError name={name} />)

    expect(wrapper.props()).toEqual({
      name: name,
      component: 'div',
      className: 'form__error-message'
    })

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
