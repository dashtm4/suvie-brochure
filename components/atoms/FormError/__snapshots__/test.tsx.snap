// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormError /> should has correct props 1`] = `
<FormikConnect(ErrorMessageImpl)
  className="form__error-message"
  component="div"
  name="test"
/>
`;
